common/src/newBilling/java/com/revenuecat/purchases/common/purchaseExtensions.kt

```diff
@@ -14,21 +14,12 @@ fun Purchase.toHumanReadableDescription() =
  * we assume that there will be only a single sku in the purchase.
  * https://android-developers.googleblog.com/2021/05/whats-new-in-google-play-2021.html
  */
-val PurchaseHistoryRecord.sku: String
-    get() = skus[0].also {
-        if (skus.size > 0) {
-            log(LogIntent.GOOGLE_WARNING, BillingStrings.BILLING_PURCHASE_HISTORY_RECORD_MORE_THAN_ONE_SKU)
-        }
-    }
-
-/**
- * **Important**: until multi-line subscriptions are released,
- * we assume that there will be only a single sku in the purchase.
- * https://android-developers.googleblog.com/2021/05/whats-new-in-google-play-2021.html
- */
-val Purchase.sku: String
+val Purchase.firstSku: String
     get() = skus[0].also {
         if (skus.size > 0) {
             log(LogIntent.GOOGLE_WARNING, BillingStrings.BILLING_PURCHASE_MORE_THAN_ONE_SKU)
         }
     }
+
+val Purchase.listOfSkus: ArrayList<String>
```

Comment:
❤️

Reply:
Glad you like the name because I didn't know what to name it haha. We need something different than `skus` so it doesn't collide 🤷‍♂️